modules/osdk-crd-templates.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="osdk-crds-templates_{context}"]
+== CRD templates
+
+Users of your Operator will need to be aware of which options are required
+versus optional. You can provide templates for each of your CRDs with a minimum
```

Comment:
I know this convo came up. Don't recall what/if we decided. 
Do we want to spell out (SO) then parenthesize the acronym in each module or just the assembly?
So, in the new modules, would you need _Custom Resource Definitions (CRDs)_?

Reply:
I've been doing it by modules.